packages/core/components/VisxShape.tsx

```diff
@@ -21,23 +17,41 @@ interface VisxShapeProps {
   left: number
   top: number
   display: boolean
-  index: GlyphKeys
+  index: number
   stroke: string
   key: string | number
+  strokeWidth?: number
 }
 
-const VisxShape: React.FC<VisxShapeProps> = ({ fill, size, left, top, display, index, stroke, key }) => {
+const VisxShape: React.FC<VisxShapeProps> = ({
```

Comment:
Since visx handles these properties already it seems redundant to use a separate file here.

Reply:
I removed file.